content/en/database_monitoring/setup_mongodb/mongodbatlas.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Setting Up Database Monitoring for MongoDB Atlas
+kind: documentation
+description: Install and configure Database Monitoring for MongoDB Atlas
+further_reading:
+- link: "/integrations/mongo/"
+  tag: "Documentation"
+  text: "Basic MongoDB Integration"
+
+---
+
+Database Monitoring provides deep visibility into your MongoDB databases by exposing database metrics, operation samples, explain plans and events.
+
+Do the following steps to enable Database Monitoring with your database:
+
+1. [Grant the Agent access to your MongoDB Atlas Cluster](#grant-the-agent-access-to-your-mongodb-atlas-cluster)
+2. [Install and configure the Agent](#install-and-configure-the-agent)
+
+## Before you begin
+
+Supported MongoDB major versions
+: 4.4, 5.0, 6.0, 7.0
+
+{{% dbm-mongodb-before-you-begin %}}
+
+## Grant the Agent access to your MongoDB Atlas Cluster
+
+The Datadog Agent requires read-only access to the MongoDB Atlas Cluster in order to collect statistics and queries.
+
+### Create custom monitoring role
+
+1. In the MongoDB Atlas UI, navigate to the `Database Access` tab.
+2. Click tab `Custom Roles` and then `Add New Custom Role`.
+3. Enter `Custom Role Name`, e.g. `datadog`.
+4. Add the following permissions to the custom role:
+   - `read` on the `admin` database
+   - `read` on the `local` database
+   - `clusterMonitor` on the `admin` database
+   - `read` on the databases you want to monitor OR `readAnyDatabase` to monitor all databases
+5. Click `Add Custom Role`.
+
+### Create a monitoring user with the custom monitoring role
+
+1. In the MongoDB Atlas UI, navigate to the `Database Access` tab.
+2. Click tab `Database Users` and then `Add New Database User`.
+3. Under the `Authentication Method`, select `Password`.
+4. Enter the `Username` and `Password`.
+5. Under `Database User Privileges`, expand `Custom Roles` and select the custom monitoring role you created in the previous step.
+6. Click `Add User`.
+7. Note the `Username` and `Password` for the monitoring user as you will need them to configure the Agent.
+
+## Install and configure the Agent
+
+To monitor your MongoDB Atlas Cluster, you need to install and configure the Datadog Agent on a host that can remotely access your MongoDB Atlas Cluster. This host can be a Linux host, a Docker container, or a Kubernetes pod.
+
+### Get the individual MongoDB instance hostname and port from the SRV connection string
+
+Applications usually connect to MongoDB Atlas using an SRV connection string. However the Datadog Agent must connect directly to the individual MongoDB instance being monitored. If the Agent connects to different MongoDB instance while it is running (as in the case of failover, load balancing, and so on), the Agent calculates the difference in statistics between two hosts, producing inaccurate metrics.
```

Comment:
Applications usually connect to MongoDB Atlas using an SRV connection string, but **the Datadog Agent must connect directly to the individual MongoDB instance being monitored.**